github/Organization.py

```diff
@@ -541,30 +541,32 @@ def create_secret(
             completed=False,
         )
 
-    def get_secrets(self) -> PaginatedList[OrganizationSecret]:
+    def get_secrets(self, secret_type: str = "actions") -> PaginatedList[OrganizationSecret]:
         """
         Gets all organization secrets
+        :param secret_type: string options actions or dependabot
         :rtype: :class:`PaginatedList` of :class:`github.OrganizationSecret.OrganizationSecret`
         """
         return PaginatedList(
```

Comment:
We should restrict the accepted values for `secret_type` here:
 ```suggestion
         assert secret_type in ["actions", "dependabot"], secret_type
         return PaginatedList(
```

Reply:
@EnricoMi  done, also added unittest to cover the assert  please recheck